tests/test_derived_datasets.py

```diff
@@ -102,13 +108,15 @@ def test_create_derived_dataset_config(tmp_path):
     orig_df = read_dataframe(REGISTRY_PATH / "data" / dataset_id / "1.0.0" / "table.parquet")
     new_df = read_dataframe(query_output / "table.parquet")
     assert sorted(new_df.columns) == sorted(orig_df.columns)
+
+    # orig_df does not have time-wrapping, so need to load project_time to convert
+    project_time_dim = project.config.get_base_dimension(DimensionType.TIME)
+    orig_df = project_time_dim.convert_dataframe(orig_df, project_time_dim)
```

Comment:
This seems awkward. After this branch is merged in, if we remade the table.parquet, would it already be in project time? If so, maybe we need to add a TODO to update the original data sometime soon?